code/controllers/subsystem/ticker.dm

```diff
@@ -247,6 +247,7 @@ SUBSYSTEM_DEF(ticker)
 	current_state = GAME_STATE_PLAYING
 	Master.SetRunLevel(RUNLEVEL_GAME)
 	SSatoms.lateinit_roundstart_atoms()
+	SSatoms.spawn_static_comms()
```

Comment:
Make your own subsystem (or reuse init of another), don't strap things in SSatoms

Reply:
Sorry this is roundstart

Just setup a signal then, but really, don't strap random stuff in SSatoms

Reply:
geev fix